src/compute/src/server.rs

```diff
@@ -114,16 +113,6 @@ pub async fn compute_node_serve(
     .await
     .unwrap();
 
-    // A hummock compactor is deployed along with compute node for now.
-    if let Some(hummock) = state_store.as_hummock_state_store() {
-        sub_tasks.push(Compactor::start_compactor(
-            hummock.inner().options().clone(),
```

Comment:
We may also remove the inner method. 😄

Reply:
Still used in ssbench. The call site will be refactored and then we can remove inner.